src/index.tsx

```diff
@@ -328,13 +334,283 @@ export class App extends React.Component<{}, AppState> {
     }
   };
 
+  private renderSelectedElementsPopover(
+    elements: readonly ExcalidrawElement[]
+  ) {
+    const selectedElements = elements.filter(el => el.isSelected);
+    if (selectedElements.length === 0) {
+      return null;
+    }
+
+    const x = Math.max(
+      ...selectedElements.map(el => (el.width > 0 ? el.x + el.width : el.x))
+    );
+    const y = Math.min(
+      ...selectedElements.map(el => (el.height < 0 ? el.y + el.width : el.y))
+    );
+
+    return (
+      <div
+        style={{
+          left: x + this.state.scrollX + 16,
+          top: y + this.state.scrollY - 4,
+          position: "absolute",
```

Comment:
Please use css for those

Reply:
Sure. It is just a draft PR. Inline styles helped me to quickly prototype the idea which I wanted to show in this pr :) I will cleanup all the code